ios/NotificationsHelper.swift

```diff
@@ -12,18 +12,24 @@ import UIKit
   static var enabled: Bool = true
   
   @objc class func register() {
-    let notificationsSettings = UIUserNotificationSettings(forTypes: [.Badge], categories: nil)
+    let notificationsSettings = UIUserNotificationSettings(forTypes: [.Alert, .Badge], categories: nil)
     UIApplication.sharedApplication().registerUserNotificationSettings(notificationsSettings)
   }
   
-  class func updateBadge(num: Int) {
+  class func updateNotifications() {
     if (enabled) {
-      UIApplication.sharedApplication().applicationIconBadgeNumber = num;
+      UIApplication.sharedApplication().cancelAllLocalNotifications()
+      // Clean any badge
+      UIApplication.sharedApplication().applicationIconBadgeNumber = 0
```

Comment:
Yeah it was weird. But I was getting an old value. Always, even if I uninstall the app. Don't know if it's from a previous session, but decided to keep it like that to be sure we don't show any badge.